PlateSpace/Web/src/containers/main-container/main-container.js

```diff
@@ -142,9 +144,20 @@ class MainContainer extends Component {
     MongoDbManager.logout().then(() => this.props.router.replace(`/`));
   }
 
-  render() {
-    const renderList = this.props.router.isActive({ pathname: 'restaurants' });
+ getParameterByName(name, url) {
```

Comment:
this is a little exotic. i'd suggest using a library like `querystring` to parse out url data.  Or you could use something like [this](https://stackoverflow.com/a/2091331/59438) and avoid the dependency.
Regexes are edge-casey and hard to read so I would suggest avoiding them here.